demo/compose.desktop.yaml

```
@@ -0,0 +1,29 @@
+services:
+        
+  zed:
+    image: husarion/zed-desktop:noetic
+    container_name: zed
+    network_mode: host
+    runtime: nvidia
+    tty: true
+    environment:
+      - DISPLAY
+      - QT_X11_NO_MITSHM=1
+      - NVIDIA_VISIBLE_DEVICES=all
+      - NVIDIA_DRIVER_CAPABILITIES=compute,utility,display,video
+    volumes:
+      - /tmp/.X11-unix:/tmp/.X11-unix:rw
+      - ./params:/ros_ws/src/zed-ros-wrapper/zed_wrapper/params
+    privileged: true
+    command: roslaunch zed_wrapper <camera_model>.launch
+
+  rviz:
+    image: husarion/rviz:noetic
+    network_mode: "host"
```

```suggestion
    image: husarion/rviz:noetic
    container_name: rviz
    network_mode: host
```
same applies to other compose files